Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptSearchController.java

```diff
@@ -53,28 +55,43 @@
     @Value("#{messages['INDEXERSTATUS']}")
     private String indexerStatus;
 
+    @Value("${default_page_size}")
+    private Integer defaultPageSize;
+
     @InitBinder
     private void initBinder(WebDataBinder binder) {
         binder.setValidator(validator);
     }
 
     /**
-     * This method is searches concepts a specific term and pos
+     * This method is searches concepts a specific term, pos based on the
+     * pagination details.
      * 
-     * @param req
-     *            Holds the HTTP request information
      * @param model
-     *            Generic model holder for servlet
-     * @return Returns a string value to redirect user to concept search page
-     * @throws IllegalAccessException 
+     * @param page
+     * @param sortDir
+     * @param sortColumn
+     * @param conceptSearchBean
+     * @param results
+     * @return
+     * @throws LuceneException
+     * @throws IllegalAccessException
      */
     @RequestMapping(value = "/home/conceptsearch", method = RequestMethod.GET)
-    public String search(HttpServletRequest req, ModelMap model,
+    public String search(HttpServletRequest req, ModelMap model, @RequestParam(defaultValue = "1") String page,
+            @RequestParam(defaultValue = IConceptDBManager.ASCENDING + "") String sortDir,
+            @RequestParam(defaultValue = "word") String sortColumn,
+            @RequestParam(required = false, defaultValue = "") String conceptIdsToMerge,
             @Validated @ModelAttribute("conceptSearchBean") ConceptSearchBean conceptSearchBean, BindingResult results)
                     throws LuceneException, IllegalAccessException {
         if (results.hasErrors()) {
             return "conceptsearch";
         }
+        
+        if (conceptIdsToMerge != null && !conceptIdsToMerge.equals("")) {
```

Comment:
I don't think this check is really necessary. If the attribute is null, it's the same as when it's not set I believe.